app/assets/javascripts/discourse/app/components/create-account.js

```diff
@@ -52,22 +52,20 @@ export default Component.extend({
       }
     });
 
-    $(this.element).on("click.dropdown-user-field-label", "[for]", (event) => {
-      const $element = $(event.target);
-      const $target = $(`#${$element.attr("for")}`);
-
-      if ($target.is(".select-kit")) {
+    this.element.addEventListener("click", (event) => {
+      const target = document.getElementById(event.target.getAttribute("for"));
+      if (target) {
```

Comment:
Shouldn't this be:

```suggestion
      if (target.classList.contains("select-kit")) {
```

?

Reply:
very much yes